File: AgeDistribution.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileReader;
import java.io.IOException;

public class AgeDistribution {

	public static void main(String[] args) {
		File file = new File("D:/Java Code Base/CodingQuestions/Tes.txt");
		try {
			BufferedReader br = new BufferedReader(new FileReader(file));
			String line;
			while ((line = br.readLine()) != null) {
				int number = Integer.parseInt(line);
				if (number >= 0 && number <= 2) {
					System.out.println("Still in Mama's arms");
				} else if (number > 2 && number <= 4) {
					System.out.println("Preschool Manaic");

				} else if (number > 11 && number <= 14) {
					System.out.println("Middle school");

				} else if (number >= 5 && number <= 11) {
					System.out.println("Elementary school");
				} else if (number >= 15 && number <= 18) {
					System.out.println("High school");
				}

				else if (number >= 19 && number <= 22) {
					System.out.println("College");
				} else if (number >= 23 && number <= 65) {
					System.out.println("Working for the man");
				} else if (number >= 66 && number <= 100) {
					System.out.println("The Golden Years");
				} else {
					System.out.println("This program is for humans");
				}
			}

		} catch (FileNotFoundException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		} catch (IOException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}

	}

}

File: AgeDistribution.rb
def age_distribution
	f = File.open("C:/Ruby200/Files/Tes.txt")
	f.each_line do |value|
		#puts value
		number = value.to_i
		case number
			when 0..2
				puts "Still in Mama's arms"
			when 3..4
				puts "Preschool Manaic"
			when 5..11
				puts "Elementary school"
			when  12..14
				puts "Middle school"
			when 15..18
				puts "High school"
			when 19..22
				puts "College"
			when 23..65
				puts "Working for the man"
			when 66..100
				puts "The Golden Years"
			else
				puts "This program is for humans"
		end
	end
end

age_distribution()